wagtailmodelchoosers/client/sources/ModelSource.js

```diff
@@ -1,6 +1,6 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import { DraftUtils } from 'draftail';
+import { AtomicBlockUtils } from 'draft-js';
```

Comment:
Could Draft.js be loaded from the version built into Wagtail 2.0+ (`window.DraftJS`?) instead of having it installed & bundled separately?